terraform/environments/electronic-monitoring-data/modules/dms/endpoints_rds_s3.tf

```diff
@@ -44,31 +44,31 @@ resource "aws_dms_s3_endpoint" "dms_s3_parquet_target" {
   # cdc_min_file_size                           = 32000
   # cdc_path                                    = "cdc/path"
   # compression_type                            = "NONE"
-  csv_delimiter     = ","
-  csv_no_sup_value  = "false"
+  # csv_delimiter     = ","
+  # csv_no_sup_value  = "false"
   # csv_null_value    = "null"
-  csv_row_delimiter = "\\n"
-  data_format       = "csv"
-  data_page_size    = 68000000
+  # csv_row_delimiter = "\\n"
+  data_format    = "parquet"
+  data_page_size = 2048000
   # date_partition_delimiter                    = "UNDERSCORE"
   # date_partition_enabled                      = false
   # date_partition_sequence                     = "yyyymmddhh"
   # date_partition_timezone                     = "Europe/London"
   # dict_page_size_limit                        = 1000000
-  # enable_statistics                           = true
+  enable_statistics = true
   # encoding_type                               = "plain"
   # encryption_mode                             = "SSE_S3"
   # expected_bucket_owner                       = data.aws_caller_identity.current.account_id
   # external_table_definition                   = "meta"
   # glue_catalog_generation                     = true
   # ignore_header_rows                          = 1
   # include_op_for_full_load                    = true
-  max_file_size = 64000
-  # parquet_timestamp_in_millisecond            = false
-  # parquet_version = "parquet-2-0"
+  # max_file_size = 64000
+  parquet_timestamp_in_millisecond = true
+  parquet_version                  = "parquet-2-0"
   # preserve_transactions                       = false
   # rfc_4180                                    = false
-  # row_group_length                            = 11000
+  row_group_length = 20048
```

Comment:
Unsure if this is a typo, as 20048 seems an odd size? [Apache docs](https://parquet.apache.org/docs/file-format/configurations/) suggest 512MB-1GB row group size. This value would therefore suggest we're expecting each row to be 1GB/20048 = ~52kb in size, which seems pretty large... Am I misunderstanding?